docs/source/modules/event.rst

```diff
@@ -0,0 +1,99 @@
+.. _event-module:
+
+============
+Event Module
+============
+
+.. contents::
+    :depth: 3
+
+Description
+===========
+
+The Event module allows to create listener and assign action to event. The module is using :code:`MotechEvent` to provide
```

Comment:
This "assing action to event" doesn't quite sound right. Let's make it say "create a listener, that contains an action, triggered by firing an event"